examples/threadExample/src/main/java/io/realm/examples/threads/PassingObjectsFragment.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.threads;
+
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.UUID;
+
+import io.realm.Realm;
+import io.realm.examples.threads.model.Person;
+
+public class PassingObjectsFragment extends Fragment {
+
+    private TextView textContent;
+
+    private Realm realm;
+    private Person person;
+
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_passing_objects, container, false);
+        textContent = (TextView) view.findViewById(R.id.text_content);
+        view.findViewById(R.id.start_activity).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(getActivity(), ReceivingActivity.class);
```

Comment:
I would put a comment above this to explain the pattern. One time is probably enough instead of duplicating it across 3 click listeners. Alternative it could be a class comment?